styles/Home.module.css

```diff
@@ -1,5 +1,5 @@
 .container {
-  padding: 0 2rem;
+  /* padding: 0 2rem; */
```

Comment:
Do not submit commented code, its bad practice as our version control enable us to circumvent any drawback of removing code.

Delete this class altogether if its not needed, please.